README.md

```diff
@@ -218,6 +218,22 @@ Currently, the following cases are available for this task:
 - Go
   - `plain/plain.go`: An empty function that does nothing.
 
+### Task: Code Repair
+
+Code repair is the task of repairing source code with compilation errors.
+
+For this task, we introduced the `mistakes` repository, which includes five examples of source code with compilation errors for both Java and Go.
+Each example is isolated in its own package, along with a valid test suite. We compile each package in the `mistakes` repository and provide the LLM's with both the source code and the list of compilation errors. The LLM's response is then validated with the predefined test suite.
+
+#### Cases
+
+Currently, the following cases are available for this task:
+
+- Java
+  - [`java/mistakes`](https://github.com/symflower/eval-dev-quality/tree/main/testdata/java/mistakes)
+- Go
+  - [`golang/mistakes`](https://github.com/symflower/eval-dev-quality/tree/main/testdata/golang/mistakes)
```

Comment:
Can you try to change the link to be relative? I.e. `[java/mistakes](testdata/java/mistakes)` cause I think GitHub will automatically make it a link to the folder. That way it stays correct on different branches.